src/js/helpers/pageAndSortHelper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * pageAndSortHelper.js
+ * created by Jonathan Hill 01/07/2020
+ */
+
+export const pageAndSort = (data, sortFunction, page, limit, sortDirection, sortProperty) => {
+    // protects against required props
+    if (!data || !data.length || !page || !limit) return [];
+    // protects against required sorting props
+    if (!sortFunction && !sortDirection && !sortProperty) return [];
```

Comment:
shouldn't these be ors?

```suggestion
    if (!sortFunction || !sortDirection || !sortProperty) return [];

```

Reply:
This is checking to see that if there is no sort function then a user must pass sort direction and sort property.

If they pass a sort function then we do not care what the sort direction or property is.